spec/acceptance/api/internal/applications_api_spec.rb

```diff
@@ -85,6 +85,17 @@
       # The returned data should not contain *some_param* attribute
       expect(app.to_hash).to eq application.merge(user_required: false)
     end
+
+    context 'With an application that has no state' do
```

Comment:
use downcase with specs unless referring to entities in code, concepts, etc

Reply:
Done